decidim-participatory_processes/app/serializers/decidim/participatory_processes/participatory_process_importer.rb

```diff
@@ -66,6 +69,18 @@ def import_process_group(attributes)
         end
       end
 
+      def import_participatory_process_type(participatory_process_type)
+        return if participatory_process_type.blank?
+        return if (participatory_process_type["title"]).blank?
```

Comment:
```suggestion
        return if compact_translation(participatory_process_type["title"]).blank?
```

Reply:
To avoid issues similar with the ones reported in https://github.com/decidim/decidim/issues/10689 we need to use `compact_translation` method. 

Please update your branch to fetch the latest changes :)

Reply:
Oops crossing messages! I do this asap!